ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_utils.hpp

```diff
@@ -188,6 +188,12 @@ Tensor convert_conv_weight_tensor_to_special_padding_tiled_layout(
 // Converts convolution weights to grouped layout with padded zeros
 Tensor convert_conv_weight_tensor_to_grouped_layout(const Tensor& conv_weight_tensor, uint32_t num_groups, DataType output_dtype);
 
+sliding_window::ParallelConfig determine_output_parallel_config(
```

Comment:
move this declaration next to determine_parallel_config, and consider renaming determine_parallel_config to determine_input_parallel_config

Reply:
Will do

Reply:
Done.